src/forms/Form/validators.ts

```diff
@@ -68,6 +70,8 @@ export function validateNumber(value: string): string | undefined {
 
 	if (!(!isNaN(Number(value)) && !isNaN(parseFloat(value)))) {
 		return "The value is not a number";
+	} else {
+		return;

```

Comment:
Similar comment as above. You can test and make sure the solution complies with the ticket if you put a field that's not supposed to have that error message and doesn't show it.
![image](https://user-images.githubusercontent.com/23409284/166839575-d9668ec8-b1e6-4a24-b7df-38d3b79715e8.png)